composer.json

```
@@ -12,9 +12,9 @@
         "php" : ">=7.2",
         "ext-pcntl" : "*",
         "psr/event-dispatcher": "^1.0",
-        "symfony/console" : "~4.4 || ~5.0",
-        "symfony/framework-bundle" : "~4.4 || ~5.0",
-        "symfony/yaml": "~4.4 || ~5.0",
+        "symfony/console" : "~4.4 || ~5.0 || >=5.4 || >=6.1",
+        "symfony/framework-bundle" : "~4.4 || ~5.0 || >=5.4 || >=6.1",
+        "symfony/yaml": "~4.4 || ~5.0 || >=6.1",
```

```suggestion
        "symfony/console" : "~4.4 || ~5.0 || ~6.0",
        "symfony/framework-bundle" : "~4.4 || ~5.0 || ~6.0",
        "symfony/yaml": "~4.4 || ~5.0 || ~6.0",
```

because `>=6.1` is included in `>=5.4`

and if we support `5.4` and `6.1`, then we also support `6.0`

and we don't know if we support Symfony versions that does not exist yet, so we should use `~6.0` instead of `>=6.0` because we are more confident about it

and `~5.4` is included in `~5.0`